docs/pages/machine-id/access-guides/terraform.mdx

```diff
@@ -173,6 +173,91 @@ Check your Teleport cluster, ensuring the role has been created:
 $ tctl get role/terraform-test
 ```
 
+## Example: Github Actions
+
+Let's look at this in a concrete example of running Terraform in CI using GitHub Actions.
+
+First we'll create a token and bot that allow jobs running in Actions in a specific organization,
+in a specific repository where we keep all our Terraform code, to use the Terraform role:
+
+```yaml
+---
+kind: token
+version: v2
+metadata:
+  name: github-actions-bot
+  description: Token for GitHub Actions bot for CI Terraform runs
+spec: 
+  roles:
+  - Bot
+  bot_name: github-actions
+  join_method: github
+  github:
+    allow:
+    - repository: <your org>/<your terraform repo>
+---
+kind: bot
+version: v2
+metadata:
+  name: github-actions
+spec:
+  token_id: github-actions-bot
+  roles:
+  - terraform
+```
+
+Now we'll combine the available `teleport-actions` and `hashicorp` Actions to run Terraform in CI, using
+tbot to authenticate with the Teleport cluster. The Actions `teleport-actions/setup` and `teleport-actions/auth`
+perform the steps we did in Step 2 above, configuring `tbot` and then starting it up in
+[one-shot mode](../../includes/machine-id/daemon-or-oneshot.mdx).
+
+```yaml
+jobs:
+  terraform:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+    steps:
+      - name: Install Teleport
+        uses: teleport-actions/setup@v1
+        with:
+          version: (=teleport.version=)
+      - name: Authorize against Teleport
+        id: auth
+        uses: teleport-actions/auth@v2
+        with:
+          # Specify the publically accessible address of your Teleport proxy.
```

Comment:
```suggestion
          # Specify the publicly accessible address of your Teleport proxy.
```

https://www.grammarphobia.com/blog/2015/08/publicly-publically.html

Reply:
Hah, that's copy pasta from our own doc code on the Actions so gotta fix that somewhere else, I'll find it and note to do later.

Reply:
Removing this whole addition for now at Paul's suggestion. Resolving.